### PR TITLE
Updating istio version for Alpha release 0.1.0

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

**Comment by rshriram** [comment]
Presubmit failing. @sebastienvas let's not create a 0.1 release artifact yet. I suggest creating an 0.1-alpha artifact as we want a 0.1 release in 2 weeks. In this interim period, we can fix bugs, issues in docs etc.

**Comment by rshriram** [comment]
I don't remember the conclusion of the discussion today. But I would also hold off on cutting release branches until this weekend. For all we know, we might be fixing lots of bugs in all repos for next two weeks before 0.1 release

File: istio.VERSION [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # DO NOT EDIT THIS FILE MANUALLY instead use
 # tests/updateVersion.sh (see tests/README.md)
 export CA_HUB="docker.io/istio"
-export CA_TAG="2017-05-09-17.26.56"
+export CA_TAG="0.1.0"
 export MIXER_HUB="docker.io/istio"
-export MIXER_TAG="2017-05-09-20.54.56"
-export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/stable-621b2c4/artifacts/istioctl"
+export MIXER_TAG="0.1.0"
+export ISTIOCTL_URL="https://storage.googleapis.com/istio-release/releases/alpha-0.1.0/istioctl"
```
Comment (rshriram) [inline istio.VERSION]: Is this where we will be downloading the istioctl binary from? 
It should be 0.1.0-alpha1. Not the other way around. 

All images need to be tagged as 0.1.0-alpha1. If we have to update images, then we can bump it to 0.1.0-alpha2, etc. this is a temporary process for next two weeks only.

Comment (rshriram) [inline istio.VERSION]: I was under the impression that the istioctl tool will also be hosted in Github release

**Comment by sebastienvas** [comment]
Too late

File: istio.VERSION (hunk 1 above)
Comment (sebastienvas) [inline istio.VERSION]: I wish I add this information before (I asked multiple time the version of this release). The image are already tagged and the tags are pushed in the repos (auth, manager, mixer). This is just a number we can have the first production release be 0.2.0. We can also change naming scheme after.

Comment (rshriram) [inline istio.VERSION]: It just hit me, as our initial release is slated to be a 0.1. I guess its okay. 0.2.0 is still fine. But lets keep in mind that for next two weeks, we will iterate on 0.1.*, and then have 0.2.0 docker tags

Comment (sebastienvas) [inline istio.VERSION]: I think your proposal makes sense. That s actually a common scheme. We an switch after. I am still investigating the failure.

**Pull request merged**